internal/user/repository/view/model/user_session.go

```diff
@@ -106,8 +116,13 @@ func (v *UserSessionView) AppendEvent(event *models.Event) {
 	case es_model.MFAOTPCheckFailed,
 		es_model.MFAOTPRemoved,
 		es_model.HumanMFAOTPCheckFailed,
-		es_model.HumanMFAOTPRemoved:
+		es_model.HumanMFAOTPRemoved,
+		es_model.HumanMFAU2FTokenCheckFailed: //TODO: token removed?
```

Comment:
Handle U2f removed and passwordless removed

Reply:
removed has been added